docs/configuration.md

```diff
@@ -124,6 +124,20 @@ the value. For example, `listen_address = 127.0.0.1:$ID` makes each
              The path of the metric plugin dynamic library to load during runtime.
   --metric_plugin_config PATH
              The path to the configuration file of the plugin.
+  --tls_certificate_file CERTFILE
+             The path of a file containing the TLS certificate and CA
+             chain.
+  --tls_ca_file CERTAUTHFILE
+             The path of a file containing the CA certificates chain.
+  --tls_key_file KEYFILE
+             The path of a file containing the private key corresponding
+             to the TLS certificate.
+  --tls_password PASSCODE
+             The passphrase of encrypted private key
```

Comment:
TLS_PASSWORD as an environment variable still works, right?  Most people pass secrets via environment variables, as I think command line leaks more places.

Reply:
@alexmiller-apple ping

Reply:
I think it's fair to leave things as they are here.  I just wanted to make sure that FDB_TLS_PASSWORD would still work, and we should probably clarify the [tls docs](https://apple.github.io/foundationdb/tls.html?highlight=tls) about (1) that --tls_flag overrides FDB_TLS_FLAG (2) FDB_TLS_PASSWORD is likely preferred over --fdb_tls_password for security reasons.

I'll go file a bug for that doc change, but you're welcome to go do it instead. :)

Reply:
Opened issue [here](https://github.com/apple/foundationdb/issues/984). @alexmiller-apple